### PR TITLE
Feature/pro 146

### PR DESCRIPTION
**Pull request by bibek-maxdrive**
vNuban creation completed

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -36,6 +35,7 @@ app.get('/', (request, response) => {
 });
 
 app.get('/create-contracts', createContractController);
+app.post('/getchamps', getChamps);
```
Comment (rudrameher) [inline app.js:38]: please put proper name.`get-champion/s`

File: controller/createVNubanController.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+  }
+ 
+  let url1 = `${appConfig.NOTIFICATION_PUSH_URL}/v1/email/send`
+  axios.post(url1, emailBody, {headers :{
+    'Content-Type': 'application/json',
+    'Authorization': `Bearer ${appConfig.authToken}`
+  }
+})
+  .then(function (response) {
+    //console.log(response);
```
Comment (rudrameher) [inline controller/createVNubanController.js:26]: use async await to remove this callbacks. also remove commented lines. use try catch

File: controller/createVNubanController.js [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+  axios.post(url, emailBody,{headers :{
+    'Content-Type': 'application/json',
+    'Authorization': `Bearer ${appConfig.authToken}`
+  }
+  }  )
+  .then(function (response) {
+    //console.log(response);
+  })
+  .catch(function (error) {
+    console.log('Error sending Email ', error);
```
Comment (rudrameher) [inline controller/createVNubanController.js:64]: same here

File: cronjob.js [hunk 4]
```diff
@@ -94,6 +95,14 @@ const initCronJob = () => {
       postSlackMessage(message);
     }
   });
+
+  const jobs = schedule.scheduleJob('0 9-21/2 * * 1-6', async () => {
```
Comment (rudrameher) [inline cronjob.js:99]: assign this cron timing to a variable and add  a comment for the developers to know what is the exact time of run

File: utils/objection-error-handler.ts [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+      data: {},
+    });
+  } else if (err instanceof ConstraintViolationError) {
+    res.status(404).send({
+      message: err.message,
+      type: 'NotFound',
+      data: {},
+    });
+  } else if (err instanceof UniqueViolationError) {
+    res.status(409).send({
```
Comment (rudrameher) [inline utils/objection-error-handler.ts:94]: use a switch case and remove hardcoded status code

**Review by rudrameher — COMMENTED** [review]
Also this is a plain javascript code. if you are using typescript then please start moving other files to typescript

**Comment by bibek-maxdrive** [comment]
PR created from new branch feature/PRO-146-create-vnuban

**Pull request closed without merging**